know/rdf/parser/summarizer.py

```diff
@@ -29,3 +32,18 @@ def valid_viaf(self):
             Otherwise returns false
         """
         return 'viaf' in self.uri
+
+    def get_wikidata_uri(self):
+        """ Parse viaf rdf file and search for a wikidata
+            resource that is the 'sameAs' the given one.
+        """
+        graph = rdflib.Graph()
+        graph.parse(self.uri)
+        viaf_uri_base = "http://viaf.org/viaf/"
+        entity_id = re.search('[0-9]+', self.uri).group(0)
+        viaf_entity = rdflib.URIRef(viaf_uri_base + entity_id)
+        schema = Namespace('http://schema.org/')
+        for _, __, obj in graph.triples((viaf_entity, schema.sameAs, None)):
+            if 'wikidata' in str(obj):
+                return str(obj)
+        raise Exception("Couldn't find associated wikidata URI!")
```

Comment:
```suggestion
        graph = rdflib.Graph()
        graph.parse(self.uri)

        entity_id = re.search('[0-9]+', self.uri).group(0)
        viaf_uri_base = "http://viaf.org/viaf/"

        viaf_entity = rdflib.URIRef(viaf_uri_base + entity_id)
        schema = Namespace('http://schema.org/')

        for _, __, obj in graph.triples((viaf_entity, schema.sameAs, None)):
            if 'wikidata' in str(obj):
                return str(obj)

        raise Exception("Couldn't find associated wikidata URI!")
```
not having blank lines for separation makes this much harder to read imo